clang/lib/StaticAnalyzer/Core/RegionStore.cpp

```diff
@@ -67,9 +67,10 @@ class BindingKey {
             isa<ObjCIvarRegion, CXXDerivedObjectRegion>(r)) &&
            "Not a base");
   }
-public:
 
+public:
   bool isDirect() const { return P.getInt() & Direct; }
+  bool isDefault() const { return !isDirect(); }
```

Comment:
```suggestion
```
This is no longer needed -- in the "sort key" reformulation it was more natural to use `isDirect()`.